docs/deploy-cluster.md

```diff
@@ -62,7 +62,6 @@ Property Name               | Default Value            | Description
 `pid_file`                  | "pids/nebula-metad.pid"  | File to hold the process id.
 `daemonize`                 | true                     | Whether run as a daemon process.
 `cluster_id`                | 0                        | A unique id for each cluster.
```

Comment:
IIRC,  "cluster_id" is removed again,  we have one flag "cluster_id_path" which is used to save clusterId on this path.